plugin/rewrite/rewrite.go

```diff
@@ -33,34 +33,29 @@ const (
 type Rewrite struct {
 	Next     plugin.Handler
 	Rules    []Rule
-	noRevert bool
+	noRevert bool // for test
```

Comment:
oh members that _are_ only used for testing, that sounds iffy. Can this be done in another why. I would very much not have this done this way

Reply:
The `noRevert` was already there, it has not been introduced by this PR. I just added another such field, and added the comment telling the reason for this field. I just found no other usage for the already existing field, besides various test cases.